ci/build_container/build_container_ubuntu.sh

```diff
@@ -12,6 +12,7 @@ wget -O - http://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
 apt-add-repository "deb http://apt.llvm.org/xenial/ llvm-toolchain-xenial-7 main"
 apt-get update
 apt-get install -y clang-7 clang-format-7 clang-tidy-7 lld-7
+apt-get install -y libc++-7-dev libc++abi-7-dev
```

Comment:
use the same line above?

Reply:
tools vs library, but I don't care too much either way...